lib/http/response/index.js

```diff
@@ -23,7 +24,7 @@ module.exports = function onResponse(res) {
     if (statusCode >= 400) {
       res.destroy();
       var he = new error.HttpError(statusCode);
-      he.url = res.req && res.req.path;
+      he.url = res.req && url.format(res.req);
```

Comment:
you can add a test for this I think
https://github.com/CondeNast/copilot-util/blob/master/test/test-http-response.js#L51